addons/product/models/product_pricelist_item.py

```diff
@@ -104,6 +104,8 @@ def _default_pricelist_id(self):
             ('formula', "Formula"),
             ('fixed', "Fixed Price"),
         ],
+        help="Use the discount rules and activate the discount settings"
+             "in order to show discount to customer.",
```

Comment:
```suggestion
             " in order to show discount to customer.",
```
space probably needed here